test/unit/io/sam_file/format_sam_test.cpp

```diff
@@ -188,24 +188,36 @@ TEST_F(sam_format, header_errors)
             "@HD\tGO:none\tSO:coordinate\tVN:1.6\tSS:coordinate:queryname\n"
             "@PG\tPN:novoalign\tPP:qc\tID:novoalign\tVN:V3.02.07\tCL:novoalign -d /hs37d5.ndx -f /file.fastq.gz\n"
             "@SQ\tAS:hs37d5\tSN:ref2\tLN:243199373\n"
-            "@RG\tLB:1\tSM:NA12878\tID:U0a_A2_L1\tPL:illumina\tPU:1\n"
+            "@RG\tLB:1\tSM:NA12878\tPL:illumina\tPU:1\tID:U0a_A2_L1\n"
         };
         std::istringstream istream(header_str);
         seqan3::sam_file_input fin{istream, seqan3::format_sam{}};
         EXPECT_NO_THROW(fin.begin());                       // Order of tags should not trigger errors
     }
-    // user defined tags
-    {
+    // user defined tags -> cerr of unspported tags
+    {
+        /* User defined tags should not trigger errors,but prints in cerr:
+            Not supported SAM header tag in @HD: VB
+            Not supported SAM header tag in @HD: SB
+            Not supported SAM header tag in @HD: GB
+            Not supported SAM header tag in @HD: pb
+            Not supported SAM header tag in @PG: IB
+            Not supported SAM header tag in @PG: PB
+            Not supported SAM header tag in @PG: CB
+            Not supported SAM header tag in @PG: DB
+            Not supported SAM header tag in @PG: VB
+            Not supported SAM header tag in @PG: pb
+        */
         std::string header_str
         {
-            "@HD\tVN:1.6\tSO:coordinate\tSS:coordinate:queryname\tGO:none\tpb:usr_defined_tag\n"
-            "@PG\tID:qc\tPN:quality_control\tCL:qc -f file\tDS:trim reads with low qual\tVN:1.0.0\tpb:usr_defined_tag\n"
-            "@SQ\tSN:ref2\tLN:243199373\tAS:hs37d5\tpb:usr_defined_tag\n"
-            "@RG\tID:U0a_A2_L1\tPL:illumina\tPU:1\tLB:1\tSM:NA12878\tpb:usr_defined_tag\n"
+            "@HD\tVN:1.6\tVB:usr_defined_tag\tSB:usr_defined_tag\tGB:usr_defined_tag\tpb:usr_defined_tag\n"
```

Comment:
Can you check if something like here works?

https://github.com/seqan/seqan3/blob/929c355bd14caeb9761996b25dc9271379da776b/test/unit/argument_parser/format_parse_test.cpp#L24-L26 (but with `EXPECT_EQ([...]GetCapturedStderr(), "Unsupported...")`

Reply:
jepp that works!